### PR TITLE
Align with the OSC Documentation Policy

### PR DESCRIPTION
**Pull request by zeme-wana**
Add top-level files in alignment with the OSC Documentation Policy.
See https://github.com/IntersectMBO/plutus/issues/6423

### CODE REVIEW

File: CODE_OF_CONDUCT.md
Comment (zeme-wana) [inline CODE_OF_CONDUCT.md:1]: Taken from https://github.com/IntersectMBO/OSC-documentation/blob/master/CODE_OF_CONDUCT.md

File: SECURITY.md
Comment (zeme-wana) [inline SECURITY.md:1]: Taken from https://github.com/IntersectMBO/OSC-documentation/blob/master/SECURITY.md

File: DESCRIPTION.md
Comment (zeme-wana) [inline DESCRIPTION.md:1]: The [OSC governance policy](https://intersect.gitbook.io/open-source-committee/policies/governance/documentation) expects a DESCRIPTION file. 
However, this is redundant as all information is already included in the README file.
Also, the [OSC Documentation Repo](https://github.com/IntersectMBO/OSC-documentation) itself doesn't have a DESCRIPTION file.
I'm unsure what to do here. Is the DESCRIPTION file really necessary?

File: LICENSE
Comment (zeme-wana) [inline LICENSE:1]: Replaces with [this license](https://intersect.gitbook.io/open-source-committee/policies/governance/legal) as per the policy. It's the same contents, just different formatting

File: LICENSE.md
Comment (zeme-wana) [inline LICENSE.md:1]: Taken from https://intersect.gitbook.io/open-source-committee/policies/governance/legal

File: CHANGELOG.md
Comment (ana-pantilie) [inline CHANGELOG.md:8]: Unrelated to this PR, I find it strange that we also release `prettyprinter-configurable` but it doesn't have a changelog. @IntersectMBO/plutus-core-maintainers any idea why this is?

**Review by ana-pantilie — DISMISSED** [review]
LGTM, should @zliu41 also approve before merging?

File: CODEOWNERS.md
Comment (ana-pantilie) [reply CODEOWNERS.md:1]: I just realised something: this will block merging unless it gets reviewed by the specific code owner, right? Since you're the only code owner for the nix code, it might make sense to add someone else as well in case you're unavailable.

Comment (zeme-wana) [reply CODEOWNERS.md:1]: Interesting questions, I'm not sure. 
I also don't know how CODEOWNER rules will play with existing PR & Merge rules.
We can just try it like this and see what happens?

File: CHANGELOG.md
Comment (effectfully) [reply CHANGELOG.md]: We hope to extract it into a separate library and publish it on Hackage some time, so that we don't need to keep it in the Plutus repo at all.

File: CODEOWNERS.md
Comment (zliu41) [reply CODEOWNERS.md:1]: Does the OSC policy require there be a code owner for everything? I doubt adding `plutus-core-maintainers` to every PR is a good idea.

**Review by zliu41 — APPROVED** [review]

**Pull request merged**
